content/tagging/_index.md

```diff
@@ -38,49 +38,44 @@ A few best practices on tags:
     * Periods
     * Slashes
 
-    Other special characters get converted to underscores.
-    **Note**: A tag cannot end with a colon (e.g., `tag:`)
 2. Tags can be **up to 200 characters** long and support unicode.
 3. Tags are converted to lowercase.
-4. A tag can have a `value` or a `key:value` syntax:
-    **For optimal functionality, we recommend constructing tags that use the `key:value` syntax.** The key is always what precedes the first colon of the global tag definition, e.g.:
+4. A tag can be in the format `value` or `key:value`. For optimal functionality, **we recommend constructing tags in the `key:value` format.** Commonly used tag keys are `env`, `instance`, `name`, and `service`. The key is always what precedes the first colon of the global tag definition, for example:
+    
+    | Tag                      | Key                | Value            |
+    |--------------------------|--------------------|------------------|
+    | `service:database:mysql` | `service`          | `database:mysql` |
+    | `service_database:mysql` | `service_database` | `mysql`          |
 
-    * `role:database:mysql` is parsed as **key**:`role` , **value**:`database:mysql`
-    * `role_database:mysql` is parsed as **key**:`role_database` , **value**:`mysql`
-
-    Examples of commonly used metric tag keys are `env`, `instance`, `name`, and `role`.
-
-5. `device`, `host`, and `source` are **reserved tag keys** and cannot be specified in the standard way.
+5.  **Reserved tag keys** `device`, `host`, and `source` cannot be used in the standard way.
```

Comment:
`service` is also a reserved tag key.. Maybe we should describe in an array why each of those tag key cannot be used in the standard way.

Reply:
Putting array in the intro section above for now and waiting for confirmation on reserved tags. We can change later if needed.